.github/dependabot.yml

```diff
@@ -1,15 +1,7 @@
 version: 2
 updates:
-- package-ecosystem: cargo
-  directory: "/"
-  schedule:
-    interval: weekly
-  open-pull-requests-limit: 3
-  allow:
-  - dependency-type: direct
-  - dependency-type: indirect
```

Comment:
Is it intentional to revert commit https://github.com/rust-vmm/vhost-user-backend/commit/f363fb2b53abc13ca017ec472ac2f100be381783 ?

Reply:
Nooooo. Good catch! Let me address that.

Reply:
Done.